src/Lazy/LList.elm

```diff
@@ -236,11 +216,11 @@ This function is performed lazily.
 
 -}
 map : (a -> b) -> LList a -> LList b
-map predicate =
-    Lazy.map (List.map predicate)
+map f =
```

Comment:
predicate is commonly used to mean `a -> Bool`

Reply:
agree